service/src/main/kotlin/fi/espoo/evaka/invoicing/service/IncomeQueries.kt

```diff
@@ -68,15 +68,15 @@ WITH latest_income AS (
     WHERE :checkForExpirationRange @> i.valid_to
      AND daterange(pl.start_date, pl.end_date, '[]') @> (i.valid_to + INTERVAL '1 day')::date
 )
-SELECT guardian_id, valid_to AS expiration_date
+SELECT person_id AS guardian_id, valid_to AS expiration_date
```

Comment:
Possible alternative solution might have been to just change this to
```
SELECT DISTINCT ON(guardian_id)
guardian_id, valid_to AS expiration_date
...
ORDER BY guardian_id, valid_to
```

Reply:
If no other reason for this suggested change I'm willing to go with current as I like to explicitly do the distinct selection where it happens now